master/master.cfg

```diff
@@ -88,32 +88,27 @@ LLVM_BRANCHES = {LLVM_MAIN: VersionedBranch(ref='main', version=Version(16, 0, 0
 # (Note that there are older releases of Halide that we no longer bother to build/test regularly.)
 
 HALIDE_MAIN = 'main'
-# TODO: enable once Halide release/15.x branch created
-# HALIDE_RELEASE_15 = 'release_15'
+HALIDE_RELEASE_15 = 'release_15'
 HALIDE_RELEASE_14 = 'release_14'
 HALIDE_RELEASE_13 = 'release_13'
 
 _HALIDE_RELEASES = [
-    # TODO: enable once Halide release/15.x branch created
-    # HALIDE_RELEASE_15,
+    HALIDE_RELEASE_15,
     HALIDE_RELEASE_14,
     HALIDE_RELEASE_13,
 ]
 
-# TODO: Halide main is still v15 for now
-HALIDE_BRANCHES = {HALIDE_MAIN: VersionedBranch(ref='main', version=Version(15, 0, 0)),
-                   # TODO: enable once Halide release/15.x branch created
-                   # HALIDE_RELEASE_15: VersionedBranch(ref='release/15.x', version=Version(15, 0, 0)),
+HALIDE_BRANCHES = {HALIDE_MAIN: VersionedBranch(ref='main', version=Version(16, 0, 0)),
+                   HALIDE_RELEASE_15: VersionedBranch(ref='release/15.x', version=Version(15, 0, 0)),
```

Comment:
Should this be 15, 0, 2?

Reply:
Actually I'm generally confused about the difference between this and the llvm release versions about

Reply:
LLVM 15.0.2 is the latest release of LLVM 15.

The numbers here are the release numbers for *Halide*.